contrib/win32/openssh/OpenSSHUtils.psm1

```diff
@@ -168,7 +168,9 @@ function Repair-AuthorizedKeyPermission
                 $userProfilePath =  $properties.ProfileImagePath
             }
             $userProfilePath = $userProfilePath.Replace("\", "\\")
-            $fullPath -match "^$userProfilePath[\\|\W|\w]+authorized_keys$"
+            if ( $properties.PSChildName -notmatch '\.bak$') {
```

Comment:
@bingbing8 - Windows will also create additional, profile when users move between domains or a profile is corrupt.  These result it other profile folders that are orphaned (invalid for authorized_keys purposes).

Rather than the exclude, maybe the regex should just be tightened up to not match "...\userprofilepath<something>\...", but only "...\userprofilepath\..." ?